### ISSUE TITLE
Indent on chained methods with and without return

### ISSUE THREAD

**Issue by grimurd** [eslint/eslint#6149]
I'm using js-beautify to format my code, it works very well with ESLint except for this particular scenario.

Everytime I have a chained method such as a promise and the method is not a property of some object I get the error shown below. However if the method I'm using is a property of another object it works fine.

**What version of ESLint are you using?**
2.9.0

**What parser (default, Babel-ESLint, etc.) are you using?**
babel-eslint

**Please show your full configuration:**
http://pastebin.com/zHuG77BQ

**What did you do? Please include the actual source code causing the issue.**

```javascript
// This produces an error
class SomeClass {
    someMethod() {
        return restClient({                                 // line #234
                path: 'someurl',                             // #235
                method: 'POST'                            // #236
            })                                                       // #237
            .then(response => {
                return [client, credentials, response.entity];
            })
    }
}
```
```javascript

// This does not produce an error
class SomeClass {
    someMethod() {
        return someObject
            .restClient({                                 // line #234
                path: 'someurl',                             // #235
                method: 'POST'                            // #236
            })                                                       // #237
            .then(response => {
                return [client, credentials, response.entity];
            })
    }
}

```
**What did you expect to happen?**
I expected there'd be no error.

**What actually happened? Please include the actual, raw output from ESLint.**
```
indent Expected indentation of 12 space characters but found 16.at line 234 col 17 in file.js
indent Expected indentation of 12 space characters but found 16.at line 235 col 17 in file.js
indent Expected indentation of 12 space characters but found 16.at line 236 col 17 in file.js
indent Expected indentation of 8 space characters but found 12.at line 239 col 13 in file.js
```

**Comment by mysticatea** [comment]
Thank you for this issue.

Closing as a duplicate of #4081.
Please track that issue.
